shell.nix

```diff
@@ -5,13 +5,19 @@ pkgs.mkShell {
 
   packages = with pkgs; [
     cargo
+    rustc
     nodejs
     gcc
+    pnpm
+    git
 
     pkg-config
     cmake
     webkitgtk
     gnome.libsoup
+    librsvg
+    fmt_11
+    cacert
```

Comment:
What is this package for?

Reply:
A pure nix shell does not come with ssl certificates, which is required for cmake to fetch some things